k8s/upgrade/src/constant.rs

```diff
@@ -22,7 +35,7 @@ pub(crate) fn upgrade_image_concat(image_repo: &str, image_name: &str, image_tag
 }
 
 /// Upgrade job container image tag.
-pub(crate) const UPGRADE_JOB_IMAGE_TAG: &str = "develop";
+pub(crate) const UPGRADE_JOB_IMAGE_TAG: &str = "2.0.1";
```

Comment:
shouldn't this come from the current tag?
Can probably use utils from control-plane, might need some modifications to avoid fallback and revert back to branch name in that case.